cmd/syft/cli/commands.go

```diff
@@ -138,22 +139,33 @@ func checkForApplicationUpdate() {
 
 func logApplicationConfig(app *config.Application) {
 	versionInfo := version.FromBuild()
-	log.Infof("syft version: %+v", versionInfo.Version)
+	log.Infof("%s version: %+v", internal.ApplicationName, versionInfo.Version)
 	log.Debugf("application config:\n%+v", color.Magenta.Sprint(app.String()))
 }
 
 func newLogWrapper(app *config.Application) {
-	cfg := logger.LogrusConfig{
+	cfg := logrus.Config{
 		EnableConsole: (app.Log.FileLocation == "" || app.Verbosity > 0) && !app.Quiet,
-		EnableFile:    app.Log.FileLocation != "",
-		Level:         app.Log.LevelOpt,
-		Structured:    app.Log.Structured,
```

Comment:
Does this field have to be set later if `app.Log.Structured` is `true`

Reply:
This might have been force pushed away - let me check again

Reply:
it does -- we only key off of a boolean from the app config, but the logger config needs a different formatter for this to work (the adapter doesn't have an opinion about the formatters, only allows folks to override them)

Reply:
unless we moved this "structured" boolean to the logging adapter to `anchore/go-logger`. I'm trying to leave those adapters as opinionless as possible (to make it more flexible to the caller)

Reply:
> So under app.Log.Structured on line 153 should we also update cfg.Sructured to be true?

Today there is no `Structured` element on the config (unless we do https://github.com/anchore/syft/pull/1279#discussion_r1001884788):
https://github.com/anchore/go-logger/blob/main/adapter/logrus/logger.go#L23-L29